doc/examples/applications/plot_face_detection.py

```diff
@@ -45,14 +45,14 @@
 
 ``min_neighbor_number`` and ``intersection_score_threshold`` parameters are
 made to cluster the excessive detections of the same face and to filter out
-false detections.  True faces usually has a lot of detections around them and
+false detections.  True faces usually have a lot of detections around them and
 false ones usually have single detection. First algorithm searches for
 clusters: two rectangle detections are placed in the same cluster if the
 intersection score between them is larger then
 ``intersection_score_threshold``. The intersection score is computed using the
 equation (intersection area) / (small rectangle ratio). The described
 intersection criteria was chosen over intersection over union to avoid a corner
-case when small rectangle inside of a big one have small intersection score.
+case when small rectangle inside of a big one has small intersection score.
```

Comment:
This gallery example would definitely need some proofreading... (see also "criteri*a* was chosen" above) This would deserve its own issue/PR as well.